### PR TITLE
Adding root dir question to cli

### PR DESCRIPTION
**Pull request by gauntface**
R: @jeffposnick 

First step towards #82

Some minor changes to the flow. The CLI detects directories in the current working directory and offers them in a list, otherwise the user can manually enter in a directory name.

This is the same code as previous PR - I just re-did to avoid weird merge conflict / output from Github.

### CODE REVIEW

File: packages/sw-cli/src/cli/errors.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+module.exports = {
```
Comment (jeffposnick) [inline packages/sw-cli/src/cli/errors.js]: It's more in keeping with the other packages to put this in a `lib/` directory.

File: packages/sw-cli/src/cli/index.js [hunk 2]
```diff
@@ -121,8 +123,113 @@ class SWCli {
...
+        resolve(directoryContents);
+      });
+    })
+    .then((directoryContents) => {
+      return directoryContents.filter((directoryContent) => {
+        return fs.statSync(directoryContent).isDirectory();
+      });
+    })
+    .then((subdirectories) => {
+      const blacklistDirectoryNames = [
```
Comment (jeffposnick) [inline packages/sw-cli/src/cli/index.js]: Consider pulling this out into a `constants.js` module.

File: packages/sw-cli/src/cli/errors.js (hunk 1 above)
Comment (gauntface) [inline packages/sw-cli/src/cli/errors.js]: Done.

File: packages/sw-cli/src/cli/index.js (hunk 2 above)
Comment (gauntface) [inline packages/sw-cli/src/cli/index.js]: Done.

**Pull request merged**